task/sonatype/0.1/samples/sample-pipeline-nexus-lifecycle-scan.yml

```diff
@@ -0,0 +1,76 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: sample-pipeline-nexus-lifecycle-scan
+spec:
+  tasks:
+    - name: git-clone
+      params:
+        - name: url
+          value: 'https://github.com/koraytugay/path-history'
```

Comment:
I wonder if there's a sample generic public project that would be better to include.